pkg/common-controller/framework_test.go

```diff
@@ -1494,3 +1501,5 @@ func emptyDataSecretAnnotations() map[string]string {
 		utils.AnnDeletionSecretRefNamespace: "default",
 	}
 }
+
+func toStrPointer(str string) *string { return &str }
```

Comment:
where is this function used? seems not in this PR

Reply:
Ah yeah, I was using it for a test originally but removed it's usage. Will remove this